src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx

```diff
@@ -14,6 +14,7 @@ import {
   getBrowserCogTrackList,
```

Comment:
The typing of this file is horrible — too many anys for no apparent reason. I understand that it's a different concern from adding analytics, but perhaps since we are changing this file anyway, fix its types as well?

Reply:
This will be done in the PR https://github.com/Ensembl/ensembl-client/pull/146